### DISCUSSION TITLE
2 and more selects

### DISCUSSION THREAD

**Discussion by ZekyTheWolf** [janosh/svelte-multiselect#214] [Q&A] [answered]
I kind have question if its possible to have more then 1 multiselect on same component....

I try that, but no luck...

**Comment by janosh** [comment, marked as answer]
With "on the same component", do you mean on the same page? That should cause no trouble. Have a look at the filter bar at the top of this page featuring 3 `<MultiSelect>` instances.

https://awesome-sveltekit.janosh.dev
